web/scripts/embed.dart

```diff
@@ -4,8 +4,8 @@
 
 // Currently not in use
 
-import 'package:dart_pad/embed/embed.dart' as embed;
-import 'package:dart_pad/polymer/polymer.dart';
+import '../../lib/embed/embed.dart' as embed;
+import '../../lib/polymer/polymer.dart';
```

Comment:
Given that files are in lib, and haven't moved, I would have thought that the old references would still have worked after running pub get. However, @devoncarew knows a lot more about how this piece of machinery works that I do. Lets get his opinion on the right thing to do here.